test/helper/kubernetes/kubernetes_helper.go

```diff
@@ -28,6 +28,7 @@ func NewHelper() *Helper {
 	overrides := &clientcmd.ConfigOverrides{}
 	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(rules, overrides).ClientConfig()
 	if err != nil {
+		//lintignore:R009
```

Comment:
Is there a way we can silence these warnings by passing a flag to the tool, rather than having these ignore comments?

Reply:
Yeah, it looks like we can run `tfproviderlint -R009=false ./...` which will skip all checks for panics. But for Terraform providers, generally you want to [avoid](https://www.hashicorp.com/blog/writing-custom-terraform-providers/) all panics and os.Exit calls, and instead surface the error to Terraform. I think I'd like to create a github issue for us to investigate these calls to panic, since we might be able to error more gracefully instead. What do you think? Then we can leave these few as an exception for now, while keeping the panic-check in place for new code that's submitted.

Reply:
Oh, I just realized only one of those panics is in the provider code - the other two are just part of the acceptance test helper code. We should definitely handle the panic in `plugin.go`